ovs/lib/vdisk.py

```diff
@@ -104,12 +122,12 @@ def delete_from_voldrv(volume_id):
         :type volume_id: str
         :return: None
         """
-        vdisk = VDiskList.get_vdisk_by_volume_id(volume_id)
-        if vdisk is not None:
-            with volatile_mutex('voldrv_event_disk_{0}'.format(vdisk.volume_id), wait=20):
+        with volatile_mutex('voldrv_event_disk_{0}'.format(volume_id), wait=20):
```

Comment:
It is. But this is the time it will wait for the lock. So if e.g. 5 other threads/processes are waiting for the same lock, they add up.